python-client/giskard/datasets/base/__init__.py

```diff
@@ -5,7 +5,7 @@
 import uuid
 from functools import cached_property
 from pathlib import Path
-from typing import Dict, Optional, List, Union, Hashable
+from typing import Dict, Hashable, List, Optional, Union
```

Comment:
@Hartorn is it `black` that did the reformatiing?

Reply:
if yes, could you:
- create a separate branch from `main`
- run black on the same files as you modified in this pr
- create a separate PR with only black modifications
- once this pr is merged merge `main` into `feature/gsk-1618-run-python-tests-on-windows-builder` 

This should only add the relevant changes in the diff of this PR, otherwise it's difficult to review :/

Reply:
It must be black or isort.
Sorry about that, I will start from main with black & isort formatting, then, I will rebase.

Reply:
I will also disable my toml linter.